.travis.yml

```diff
@@ -3,6 +3,8 @@ language: node_js
 node_js:
   - "11.10"
 cache: yarn
+install:
+  - yarn install --cwd www
```

Comment:
i think this needs to be

```
  - yarn
  - yarn --cwd www
```

otherwise you don't do the install in the root?

Reply:
You're right! I pushed the change.